packages/mdl-drawer/temporary/mdl-temporary-drawer.scss

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "mdl-animation/variables";
+@import "mdl-elevation/mixins";
+@import "mdl-theme/mixins";
+
+.mdl-temporary-drawer {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  pointer-events: none;
+  box-sizing: border-box;
+  contain: strict;
+
+  @include mdl-drawer-toolbar-spacer_;
```

Comment:
since we have these same three `@include` statements within perm drawer, does it make sense to just have an `mdl-drawer-base` mixin that just does all of this, rather than 3 separate mixins?